PD-380 Update Cluster Articles (#1941)
truenas/documentation: content/_includes/ClusterWarnings.md

> SMB clusters created in TrueNAS SCALE Bluefin are not  available for cluster expansion. 

Reviewer: ```suggestion
SMB clusters created in TrueNAS SCALE Bluefin are not available for cluster expansion. 
```

> Beginning with TrueNAS SCALE Cobia, we plan to implement a method to enable new volumes for SMB cluster expansion.

Reviewer: ```suggestion
TrueNAS SCALE Cobia plans to implement a method to enable new volumes for SMB cluster expansion.
```

Using the clustering feature on a SCALE system adds some restrictions to that system:

> * You cannot create new SMB shares separately from the clustering settings.

Reviewer: ```suggestion
* Activating clustering disables individual SMB share creation on cluster member TrueNAS systems.
```

> * You cannot add the system to a different cluster.

Reviewer: ```suggestion
* Systems join a single cluster at a time.
```

> * You cannot remove single systems from one cluster and migrate to another. This is not currently supported. 

Reviewer: ```suggestion
* Removing or migrating systems from a cluster requires deleting the entire cluster.
```

> * You cannot remove a system from a cluster until you delete the entire cluster. 

Reviewer: ```suggestion
```

> * You cannot reuse an existing node without properly leaving an the cluster and cleaning up old gluster-related metadata.

Reviewer: ```suggestion
```

> Existing non-clustered SMB shares no longer function. 

Reviewer: Redundant with SMB config disable comment above.
```suggestion
```
